pkgs/development/python-modules/deepgram-sdk/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pythonOlder
+, setuptools
+, aiohttp
+, dataclasses-json
+, httpx
+, typing-extensions
+, verboselogs
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "deepgram-sdk";
+  version = "3.1.7";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  doCheck = false;
```

Comment:
What's the issue with the tests?

Reply:
I'm not sure, not getting what was failing to reproduce today and I'm actually getting a pythonRuntimeDepsCheck failure today on one module - I might've been working in a polluted environment. Let me leave this here while I PR an update for typing-extensions and then see how this builds and checks against that?

Reply:
With doCheck the test step was failing with `pop_var_context: head of shell_variables not a function context`:

```
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Running phase: pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.10.13, pytest-8.0.2, pluggy-1.4.0
rootdir: /build/deepgram-sdk-3.1.7
plugins: anyio-4.3.0
collecting ... ^Mcollected 0 items                                                              

============================ no tests ran in 0.02s =============================
/nix/store/c8dj731bkcdzhgrpawhc8qvdgls4xfjv-stdenv-linux/setup: line 1578: pop_var_context: head of shell_variables not a function context
```

Reply:
Looks like that upstream doesn't have unit tests but functional tests. At first glances I would say we can't run them without additional effort.